tests/entrypoint/database/test_add.py

```diff
@@ -9,8 +8,8 @@
 
 
 @pytest.fixture()
-def engine_sqlite(tmpdir):
-    db_url = pathlib.Path(str(tmpdir)) / "database.db"
+def engine_sqlite(tmp_path):
```

Comment:
haha, how did I not know about `tmp_path`? We should probably use it everywhere in our codebase instead of `pathlib.Path(str(tmpdir))`.

Reply:
I like `tmp_path` because it saves an `import` statement (and hopefully some time as a consequence) 😄